feed.json

```diff
@@ -2,48 +2,48 @@
 	"version": 1,
 	"whatsNewLinks": [{
 		"title": "Bot Framework Composer 1.3",
-		"description": "Learn about new features in Bot Framework Composer 1.3, including new preview features you can try today.",
+		"description": "New preview features: Form Dialogs, Orchestrator and Package Manager",
 		"url": "https://techcommunity.microsoft.com/t5/azure-ai/bot-framework-composer-1-3-is-now-available/ba-p/1996923"
 	}, {
 		"title": "Build 2020 updates",
-		"description": "Read all of the updates from this years Build conference, including Composer GA and SDK v4.9.",
+		"description": "Read all of the updates from the Build conference, including Composer GA and Bot Framework SDK v4.9.",
 		"url": "https://techcommunity.microsoft.com/t5/azure-ai/build-2020-conversational-ai-updates/ba-p/1397685"
 	}, {
-		"title": "Power Virtual Agents and Composer integration preview",
-		"description": "We are pleased to announce the Public Preview of Power Virtual Agents integration with Microsoft Bot Framework Composer. Bot makers will now be able to use Bot Framework Composer to create custom content and add it to Power Virtual Agents.",
+		"title": "Microsoft Ignite 2020: Publish to Microsoft Power Virtual Agents ",
+		"description": "Open a Power Virtual Agents topic in Bot Framework Composer, create a custom bot and publish back to Microsoft Power Virtual Agents.",
 		"url": "https://powervirtualagents.microsoft.com/en-us/blog/power-virtual-agents-integration-with-bot-framework-composer-is-available-in-public-preview/"
 	}, {
-		"title": "November 2020 - Conversational AI update",
-		"description": "Updates to the Conversational AI platform, including a new release of Bot Framework Composer and Power Virtual Agents integration.",
+		"title": "Microsoft Ignite 2020: Conversational AI",
```

Comment:
Just want to double check that the year should be 2020 and not 2021

Reply:
Yes, that's where that link goes (November 2021 is still a ways off).